code/modules/food_and_drinks/food/foods/meat.dm

```diff
@@ -79,8 +79,13 @@
 			"<span class ='notice'>[user] cuts the raw cutlet with [W]!</span>", \
 			"<span class ='notice'>You cut the raw cutlet with [W]!</span>" \
 			)
-		new /obj/item/reagent_containers/food/snacks/raw_bacon(loc)
-		qdel(src)
+		var/obj/item/reagent_containers/food/snacks/raw_bacon/bacon = new(get_turf(src))
+		if(ishuman(loc))
+			var/mob/living/carbon/human/H = loc
+			qdel(src)
+			H.put_in_hands(bacon)
+		else
+			qdel(src)
```

Comment:
```suggestion
		if(ishuman(loc))
			var/mob/living/carbon/human/H = loc
			H.put_in_hands(bacon)
		qdel(src)
```

Reply:
If the cutlet isn't deleted from the hand before the bacon is put into the hand won't it be impossible for it to go to the hand since you would have to hold a knife in the other hand to cut it? I assume it can't put something into hands that are already full, no?

Reply:
Yeah you're right, ignore this suggestion. Something to note about `qdel` however is that it puts the item into a queue to be deleted, so it's not always immediate. But I think for this situation it probably clears it out of the hand fast enough to put the bacon in.